lib/fastlane/plugin/wpmreleasetoolkit/helper/metadata_download_helper.rb

```diff
@@ -4,12 +4,17 @@
 module Fastlane
   module Helper
     class MetadataDownloader
+      AUTO_RETRY_SLEEP_TIME = 20
+      MAX_AUTO_RETRY_ATTEMPTS = 30
```

Comment:
I wonder if it wouldn't make more sense to have the `auto_retry` ConfigItem be an `Integer` that would allow to set that max-retry-attempts value?

That way we could use a value like 30 (!) for CI because we really need to be sure to end up in success, while we could use a value like 3 for repos still doing manual release, because it'd be nice not to have to `UI.confirm` the retry manually every time this happens just on the first or second try, but it if fails consistently more than 3 times we might want to wait longer when confirming manually.

Reply:
That's a cool idea. I could make that change if there is practical use for it for others or if you feel strongly that it is a better approach.

My understanding is that, this issue is mostly specific to WPAndroid & WPiOS and as you mentioned it's a temporary one. That's why I did this in the most straightforward way.

Reply:
FWIW, I don't feel strongly about it.